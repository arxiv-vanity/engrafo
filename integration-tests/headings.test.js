var utils = require("./utils");

test("basic titles and headings render correctly", done => {
  utils.expectBodyToMatchSnapshot("headings/basic.tex", done);
});

test("bold is stripped from title", done => {
  utils.expectBodyToMatchSnapshot("headings/bold-title.tex", done);
});

test("TITLES IN ALL CAPS are fixed", done => {
  utils.expectBodyToMatchSnapshot("headings/all-caps-titles.tex", done);
});
