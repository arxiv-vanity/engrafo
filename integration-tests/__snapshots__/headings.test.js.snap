// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TITLES IN ALL CAPS are fixed 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    


    <div
      class="ltx_page_content"
    >
      

      <dt-article>
        <span
          class="ltx_ERROR undefined"
        >
          \\title
        </span>
        

        <div
          class="ltx_para"
          id="p1"
        >
          

          <p
            class="ltx_p"
          >
            WHY AM I SHOUTING

          </p>
          

        </div>
        

        <section
          class="ltx_section"
          id="-at-currentID"
        >
          

          <h2
            class="ltx_title ltx_title_section"
          >
            

            <span
              class="ltx_tag ltx_tag_section"
            >
              <a
                href="#-at-currentID"
              >
                <span
                  class="ltx_ERROR undefined"
                >
                  \\thesection
                </span>
                 
              </a>
            </span>
            Urgh Stop It
          </h2>
          


          <div
            class="ltx_para"
            id="-at-currentID.p1"
          >
            

            <p
              class="ltx_p"
            >
              Hello.

            </p>
            

          </div>
          

        </section>
        

      </dt-article>
      

      <dt-appendix>
        

        


        <div
          class="l-body"
        />
        

      </dt-appendix>
    </div>
    





    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;

exports[`bold is stripped from title 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    


    <div
      class="ltx_page_content"
    >
      

      <dt-article>
        <span
          class="ltx_ERROR undefined"
        >
          \\title
        </span>
        

        <div
          class="ltx_para"
          id="p1"
        >
          

          <p
            class="ltx_p"
          >
            <span
              class="ltx_text ltx_font_bold"
              style="font-size:173%;"
            >
              Why would someone do this

            </span>
            

Hello.

          </p>
          

        </div>
        

      </dt-article>
      

      <dt-appendix>
        

        


        <div
          class="l-body"
        />
        

      </dt-appendix>
    </div>
    





    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
